pkgs/applications/networking/sniffnet/default.nix

```diff
@@ -40,6 +41,7 @@ rustPlatform.buildRustPackage rec {
     xorg.libXcursor
     xorg.libXi
     xorg.libXrandr
+    libxkbcommon
```

Comment:
```suggestion
```
this is unused right? it does not influence postFixup in any way

Reply:
My bad, I'm sorry. Thank you very much.